### PR TITLE
Don't overwrite create_environment_fn

### PR DESCRIPTION
**Pull request by psc-g**
This caused a problem for minatar environments. Also removing hardcoded cpu platform.

### CODE REVIEW

File: lifting_veil/baseline.py [hunk 1]
```diff
@@ -38,7 +36,7 @@ def create_agent(sess, environment, summary_writer=None):
     gin.clear_config()
     gin.parse_config_files_and_bindings([gin_file], gin_bindings, skip_unknown=False)
 
-    agent_runner = run_experiment.TrainRunner(LOG_PATH, create_agent, gym_lib.create_gym_environment)
```
Comment (joaogui1) [inline lifting_veil/baseline.py:41]: Don't we need to use the same pattern as [here](https://github.com/JohanSamir/rainbow_extend/blob/main/lifting_veil/main.py#L60) to use minatar and gym?

Comment (psc-g) [reply lifting_veil/baseline.py:41]: that can also work but is unnecessary, since this parameter is being set in the config files.

Comment (psc-g) [reply lifting_veil/baseline.py:41]: e.g. https://github.com/JohanSamir/rainbow_extend/blob/main/lifting_veil/Configs/dqn_asterix.gin#L51

Comment (joaogui1) [reply lifting_veil/baseline.py:41]: Oh, good point, that was only needed for offline

**Pull request merged**